### ISSUE TITLE
Introduce quality-performance balance in the preferences

### ISSUE THREAD

**Issue by takahirox** [mozilla/hubs#5467]
**Is your feature request related to a problem? Please describe.**

There are and will be some or many configurations to control the balance between quality and performance In the preferences. For example, max resolution, material quality, and panning quality. There some problems about it.

* Setting one by one may be annoying for users
* Each setting may not be so straightforward for generic (non-dev) users
* It may be not so clear for generic users what settings are for controlling quality-performance balance

**Describe the solution you'd like**

I would like to suggests

* Add "Quality-Performance" tab in the preferences
* Move the preference configuration that control the balance there
* Add "Quality-performance balance" slide bar there
* Hubs Client has presets for the configuration
* Each configuration is invisible unless a user selects custom balance

**Additional context**

* Default "Quality-Performance" balance can depend on device, for example "Quality" for desktop while  "Performance" for mobiles.

**Comment by jywarren** [comment]
Agree, this is a great idea, it is mysterious what triggers the switch to low quality and it sometimes happens unexpectedly during a screen record.

**Comment by jywarren** [comment]
Maybe related idea linking this to monitoring fps https://github.com/mozilla/hubs/issues/5389
